dev/AppNotifications/AppNotifications.idl

```diff
@@ -111,6 +111,29 @@ namespace Microsoft.Windows.AppNotifications
 
         // Gets or sets whether a Notification's pop-up UI is displayed on the user's screen.
         Boolean SuppressDisplay;
+
+        // Gets or sets the Notification Device Data
+        AppNotificationDevicesData DevicesData;
```

Comment:
>AppNotificationDevicesData DevicesData; [](http://example.com/codeflow?start=8&length=39)

This should be marked with:
[contract(AppNotificationsContract, 2)]

Reply:
Thank you ! I will make this change along with other changes ,i am planning to do.